dynamic-demo-plugin/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": ".",
     "module": "esnext",
     "moduleResolution": "node",
-    "target": "es2016",
+    "target": "es2021",
```

Comment:
@spadgett FYI - we plan to upgrade our build target to ES 2021.

We can double check this, but recent versions of all Console supported web browsers should fully support this target.

Reply:
Yes, absolutely. Let's update this. I confirmed our supported browsers will all work with es2021.